### PR TITLE
Firefox: Pass network tests

### PR DESCRIPTION
**Pull request by Meir017**
relates to #112

### CODE REVIEW

File: src/PlaywrightSharp.ProtocolTypesGenerator/ProtocolTypesGeneratorBase.cs [hunk 1]
```diff
@@ -43,5 +43,8 @@ protected void GenerateRequestDefinition(StringBuilder builder, string typeName)
 
         protected void GenerateResponseDefinition(StringBuilder builder, string typeName)
             => builder.AppendLine($"internal partial class {typeName}Response: I{Project}Response");
+
+        protected void GeneratePropertyDefinition(StringBuilder builder, string type, string name)
+            => builder.AppendLine($"[System.Text.Json.Serialization.JsonPropertyName(\"{name}\")] public {type} {name.ToPascalCase()} {{ get; set; }}");
```
Comment (kblok) [inline src/PlaywrightSharp.ProtocolTypesGenerator/ProtocolTypesGeneratorBase.cs:48]: Shouldn't this be solved by the JSON serializer?

Comment (Meir017) [reply src/PlaywrightSharp.ProtocolTypesGenerator/ProtocolTypesGeneratorBase.cs:48]: for most cases yes. in firefox we have a `base64body` which is converted to `Base64Body` and deserialized as `base64Body` which is wrong. this way we won't have potential errors in the future due to this

Comment (kblok) [reply src/PlaywrightSharp.ProtocolTypesGenerator/ProtocolTypesGeneratorBase.cs:48]: Boooo! 
Good solution 😉

**Comment by Meir017** [comment]
this can be merged. I currently have a hanging test in `ResponseTextTests` so that will be in another PR

**Pull request merged**